### DISCUSSION TITLE
Copilot doenst work with version 1.31.6194

### DISCUSSION THREAD

**Discussion by vachterberg1** [community/community#34627] [Copilot Conversations] [answered]
I am trying to use copilot in my vscode, but every time I try to use it, it shows an activation error and asks for the latest version, but when I update, it is already in the latest update available.

Log error:

`Your editor's Copilot extension, version "copilot/1.31.6194", is out of date. Please install a newer version to access this service.`

**Comment by canhhfh** [comment]
If you're using macOS, you try to install VS code insider version [here](https://code.visualstudio.com/insiders/). It can fix that error.

**Comment by davecheney** [comment, marked as answer]
@vachterberg1 you need to update VSCode then newer extensions will be available.
